crates/swc/tests/fixture/deno/deno-10014/case1/output/index.map

```diff
@@ -6,6 +6,7 @@
     "x",
     "Error"
   ],
+  "sourceRoot": "$DIR/tests/fixture/deno/deno-10014/case1/input",
   "sources": [
     "../../input/index.ts"
```

Comment:
This looks invalid. To resolve a source file, you take the concatenation of the `sourceRoot` (treating it always as a directory) and the individual `source[i]`. In this case, we'd get the result `"$DIR/tests/fixture/deno/deno-10014/case1/input" + "/" + "../../input/index.ts"` which resolves to `"$DIR/tests/fixture/deno/deno-10014/input/index.ts".